x/posts/internal/types/msgs_test.go

```diff
@@ -115,9 +115,9 @@ func TestMsgEditPost_ValidateBasic(t *testing.T) {
 		error sdk.Error
 	}{
 		{
-			name:  "Invalid post id returns error",
+			name:  "Invalid post newID returns error",
```

Comment:
I think this is a refactoring error

```suggestion
			name:  "Invalid post newID returns id",
```